services/EnsService.ts

```diff
@@ -139,6 +151,18 @@ export default class EnsService {
     return this.textRecordValues;
   }
 
+  public isBlueSkyRecordValid(record) {
+
+    // validate record in a single string with the following constraints:
+    // 1. starts with did=did:plc:
+    // 2. anything after did:plc: only alphanumeric 
+    // 3. anything after did:plc: is 24 chars long (total of 36 chars)
+    if(!record.match(/^did=did:plc:[0-9a-zA-Z]+$/) || record.length !== 36) {
```

Comment:
@makhnovskiy you can add `{24}` instead of the `+` to your regex and it would validate the exact length as well:
`/^did=did:plc:[0-9a-zA-Z]{24}$/`